modules/network-observability-ebpf-agent-alert.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-agent-alerts_{context}"]
+= Using the eBPF agent alert
+
+When the Network Observability eBPF agent hashmap table is full, either the eBPF agent processes flows with degraded performance, or the capacity limiter is triggered. When this is the case, an alert shows `NetObservAgentFlowsDropped`. If you see this alert, consider increasing the `cacheMaxFlows` in the `FlowCollector`, as shown in the following example.
```

Comment:
The first sentence is not accurate. The alert is triggered either when the hashmap is full, or when the capacity limiter is triggered. (Here it says it's always the hashmap being full and from there differentiates 2 cases)

Reply:
Okay thanks for untangling these details I mixed together 😬 I reworked this entire paragraph a little so its overall less words and the first sentence is as you suggest.